backend/jottacloud/jottacloud.go

```diff
@@ -1253,6 +1292,12 @@ func (f *Fs) Copy(ctx context.Context, src fs.Object, remote string) (fs.Object,
 	}
 	info, err := f.copyOrMove(ctx, "cp", srcObj.filePath(), remote)
 
+	// if destination was a trashed file then after a successfull copy the copied file is still in trash (bug in api?)
+	if err == nil && bool(info.Deleted) && !f.opt.TrashedOnly && info.State == "COMPLETED" {
```

Comment:
I wouldn't have thought you'd need the `bool()` around `info.Deleted` here since its underlying type is a `bool`.

Reply:
Without it: `invalid operation: err == nil && info.Deleted && !f.opt.TrashedOnly (mismatched types api.Flag and bool)`

Reply:
Well I never!